### DISCUSSION TITLE
Filters managed property does not load all

### DISCUSSION THREAD

**Discussion by jenifercerveny** [microsoft-search/pnp-modern-search#2963] [Q&A] [answered]
utilizing the Refiners webpart and attempting to access a managed property for RefinableString that I set in the search on our tenant, but the PNP > Edit refiners > Selecting the dropdown to add new does not load all available managed properties in our tenant.
![RefinerLoadIssue](https://user-images.githubusercontent.com/127989647/236887358-143a3dd4-9e75-4755-9f49-6d9348838421.jpg)

**Comment by gretchunkim** [comment]
@jenifercerveny How long have you waited for the managed property field RefinableString? Also, have you checked debug layout mode to see if the field populated, yet?  This happened to me before and I had to wait about a good one to two days for it to show up/crawled.

**Reply by afnepipe** [reply]
Hi @gretchunkim 
How can I check or enable the debug layaout? Can you give me more details, please.?

**Reply by gretchunkim** [reply]
@afnepipe It's on the layout page, on 2nd page. 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/8564035/9448db35-dd3c-4436-84ad-145e9268de3e)

**Comment by wobba** [comment, marked as answer]
Type the property you want in the combo box using the keyboard. For filters you should typically be using `RefinableXXYY` anyways. The current list also shows non-refinable properties. We might change the code to a set list of all known refinable properties, but haven't gotten around to it.
